xbmc/music/Album.h

```diff
@@ -71,6 +71,8 @@ class CAlbum
   }
 
   std::string GetArtistString() const;
+  const std::vector<std::string>& GetAlbumArtist() const;
+  const std::vector<std::string>& GetMusicBrainzAlbumArtistID() const;
```

Comment:
GetArtistString() returns artist which contains the tag.GetAlbumArtist(), to keep inline with the name GetAlbumArtist should be named GetArtist and GetMusicBrainzAlbumArtistID should be named GetMusicBrainzArtistID, but that maybe causes even more confusion? Maybe rename GetArtistString to GetAlbumArtistString to be clearer, but things like album->getalbumartiststring looks kinda ugly.